pkg/controllers/dynakube/activegate_test.go

```diff
@@ -21,17 +28,134 @@ func TestReconcileActiveGate(t *testing.T) {
 			ActiveGate: dynatracev1beta1.ActiveGateSpec{Capabilities: []dynatracev1beta1.CapabilityDisplayName{dynatracev1beta1.KubeMonCapability.DisplayName}},
 		},
 	}
-
 	t.Run("no active-gate configured => nothing happens (only call active-gate reconciler)", func(t *testing.T) {
 		dynakube := dynakubeBase.DeepCopy()
 		dynakube.Spec.ActiveGate = dynatracev1beta1.ActiveGateSpec{}
 		fakeClient := fake.NewClientWithIndex(dynakube)
+		mockActiveGateReconciler := mockcontroller.Reconciler{}
```

Comment:
```suggestion
		mockActiveGateReconciler := mockcontroller.NewReconciler(t)
```

Reply:
it's important to use New here otherwise mockery won't check if we call mocks or not.